dev-lang/lua/lua-5.1.5-r107.ebuild

```
@@ -82,7 +82,7 @@ src_prepare() {
 multilib_src_configure() {
 	# We want packages to find our things...
 	sed -i \
-		-e 's:/usr/local:'${EPREFIX}'/usr:' \
+		-e 's:/usr/local:'"${EPREFIX}"'/usr:' \
```

Can we use here `"` quotes?
```suggestion
		-e "s:/usr/local:${EPREFIX}/usr:" \
```

If you accept, please fix other `dev-lang/lua` ebuilds :)

---

done

---

err, undone, i must've messed something up, so I've dropped the Lua patch.